corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -380,6 +380,22 @@ hqDefine('cloudcare/js/utils', [
         };
     };
 
+    var submitButtonListener = function (callback) {
+        var submitButtonDisabled = sessionStorage.submitDisabled;
+        var handleSubmitButtonChange = () => {
+                callback(submitButtonDisabled);
+        };
+        return {
+            listen: function () {
+                $(window).on('load', handleSubmitButtonChange);
```

Comment:
I'm not sure I actually understand what this is listening for. Doesn't `window.load` only happen once, since web apps is essentially one big single page application?